enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+
+##### Second Rollout
+
+1. The root policy is updated.
+1. The updated policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The remaining clusters have the `rolloutStatus` of `ToApply` and keep whatever policy version, if any,
+   from the failed rollout. The root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `dev` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Updated Policy Succeeded
+
+1. The root policy is updated.
+1. The updated policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The remaining clusters have the `rolloutStatus` of `ToApply` and keep the previous policy version. The
+   root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `dev` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Updated Policy Failed
+
+1. The root policy is updated.
+1. The updated policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The remaining clusters have the `rolloutStatus` of `ToApply` and keep the previous policy version. The
+   root policy `status.rolloutStatus` value is `InProgress`.
+1. One cluster in the `dev` group stays noncompliant for the duration of the `spec.rolloutStrategy.progressDeadline` of
+   10 minutes. The root policy `status.rolloutStatus` value is `Failed`. The `dev` group still keeps the policy version
+   of the failed rollout. All remaining groups stay with the last successful version.
+
+#### Updated Policy With manualPerGroup in Order
+
+1. The root policy is updated.
+1. All clusters have the `rolloutStatus` of `ToApply` and keep the previous policy version. The root policy
```

Comment:
So if user set both dev and prod to true at the same time. The order would be dev -> prod?

Reply:
Yes, that's correct.